### PR TITLE
Added bcgov theming and header

### PR DESCRIPTION
**Pull request by Gavinok**
This resolves the second requirement for #33.

Below are some pictures for reference:

image

image

One possible change would be changing the button variant to be an outline. This would better match the buttons seen in [org-book](https://bcgov.github.io/orgbook-bc-api-docs/).

image

The current variant more closely replicates the look from the [British Columbia Government Design System](https://developer.gov.bc.ca/Design-System/Primary-Button).

image

### CODE REVIEW

**Comment by esune** [comment]
It's looking good @Gavinok !

File: OCAExplorer/src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          flex: 1
+        }}
+        />
+      </Toolbar>
+    )
+  };
+
+    return (
+      <header>
+        <AppBar position="static" sx={{ alignItems: 'flex-start', height: "100px", flex:1 , justifyContent: 'left' }}>{displayDesktop()}</AppBar>
```
Comment (amanji) [inline OCAExplorer/src/components/Header.tsx:20]: 100px might be a bit wide. Let's reduce this to maybe 64px. We should also limit the logo height to 48px.

**Review by amanji — COMMENTED** [review]
One small comment. Otherwise, it looks great!

**Comment by Gavinok** [comment]
With the modified restrictions on the header and logo size 

image

**Pull request merged**